src/commands/login.io.ts

```diff
@@ -1,6 +1,28 @@
 import prompts from "prompts";
 import { C } from "../utils/screens.js";
 
+export const promptForOrigin = async () => {
+  const ownOrigin = await prompts({
+    type: 'confirm',
+    name: 'value',
+    message: 'Would you like the use custom origin?',
```

Comment:
I would guide the users a bit more here similar to GitHub CLI. Providing a custom origin is a means to connect with Enterprise.

```bash
 ▶ gh auth login
? What account do you want to log into?  [Use arrows to move, type to filter]
> GitHub.com
  GitHub Enterprise Server
```

Reply:
Looks better now indeed:

![monokle 20231120-2](https://github.com/kubeshop/monokle-cli/assets/1061942/4b5599d2-228c-4041-9745-cbc84e51fbdb)

Fixed in https://github.com/kubeshop/monokle-cli/pull/39/commits/78dd716a705460018abdcf53d74d1517d3b3d734.